### ISSUE TITLE
After screenshotting selected area, openra stops accepting keyboard inputs

### ISSUE THREAD

**Issue by PunkPun** [OpenRA/OpenRA#20709]
The only way I found to fix it is to change to another desktop, and back again. This issue also happens when recording

**Comment by PunkPun** [comment]
This also happens when recording

**Comment by PunkPun** [comment]
I checked, this issue started happening on the second playtest, when we updated the sdl version. https://github.com/OpenRA/OpenRA/pull/20559/commits/4e9d1f5632cd614dc45b31707d9d7b534d601022

**Comment by PunkPun** [comment]
This issue started happening on the second playtest, when we updated the sdl version https://github.com/OpenRA/OpenRA/pull/20559/commits/4e9d1f5632cd614dc45b31707d9d7b534d601022 from 2.0.18 to 2.26.1

I have tested various SDL versions and here are the results
* 2.26.0 - 2.26.5 - bug present
* 2.24.0 - 2.24.2 - crash on launch, couldn't test
* 2.0.22 - works well

They changed the way they mark releases after 2.0.22, so don't worry about the large version increase. 2.0.24 became 2.24.0

The oldest working version is 2.0.22, which came out a year ago. So if we were to downgrade, we wouldn't become that much out of date. It might be possible that downgrading will also fix #20715
